vllm/worker/habana_model_runner.py

```diff
@@ -1853,6 +1793,65 @@ def _check_config(self, batch_size, seq_len, is_prompt, warmup_mode):
             logger.warning("Configuration: (%s, %s, %s) was not warmed-up!",
                            phase, batch_size, seq_len)
 
+    def create_lora_mask(self, input_tokens: torch.Tensor, lora_ids: List[int],
+                         is_prompt: bool):
```

Comment:
Please add a short help string describing 
-  Why are we creating lora mask?
-  Why we have different masks for prompt, decode, logits?
-  How are we creating this mask?